TCAT/Controllers/RouteOptionsViewController.swift

```diff
@@ -283,75 +280,76 @@ class RouteOptionsViewController: UIViewController, UITableViewDelegate, UITable
             searchTime = Date()
         }
 
-        if let time = searchTime, let startingDestination = searchFrom as? CoordinateAcceptor, let endingDestination = searchTo as? CoordinateAcceptor{
+        if let time = searchTime,
+            var startingDestination = searchFrom as? CoordinateAcceptor,
+            var endingDestination = searchTo as? CoordinateAcceptor
```

Comment:
do these need to be `vars`? just didn't see where you're modifying them but could've missed it in my skimming. if not modifying them turn it back to `let`